spoon-control-flow/pom.xml

```diff
@@ -29,8 +29,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.1</version>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>${java.src.version}</source>
+                    <target>${java.src.version}</target>
```

Comment:
Switch to maven.compiler.release flag? JDK8 is gone [https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#release](https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#release)